### PR TITLE
release 4.1.1

### PR DESCRIPTION
**Pull request by minrk**

### CODE REVIEW

**Comment by ocefpaf** [comment]
@minrk can you try to rerender this `feedstock` to see if that fixes the `Windows` build? Just type

`conda smithy rerender `

from the root directory of the `feedstock` and commit the changes.

PS: make sure you have the latest `conda-smithy` first!

**Comment by ocefpaf** [comment]
Well... now we got a consisten failure across the board :smile: 

I am investigating what is causing that. Hold on...

File: recipe/meta.yaml [hunk 1]
```diff
@@ -16,6 +16,7 @@ build:
 requirements:
   build:
     - python
+    - pip
```
Comment (ocefpaf) [inline recipe/meta.yaml]: Weird that did not fix it locally for me... But s is working here! :tada:

**Comment by ocefpaf** [comment]
Remove the `bld.bat` so Windows can also be installed using `pip`.

File: recipe/meta.yaml (hunk 1 above)
Comment (minrk) [inline recipe/meta.yaml]: Yeah, I'm a bit surprised it made a difference. It's not in other recipes with the same install command...

**Comment by ocefpaf** [comment]
Argh... Still no go on Windows :unamused: 

https://ci.appveyor.com/project/conda-forge/jupyter-core-feedstock/build/1.0.12/job/n2ocggr05bck28to#L327

I am not convinced that `conda-build-setup` is a good idea. We have `conda-smithy` for that `conda-smithy` changes are explicit. Now we have another wild card that makes debug hard...

Anyway. I send a PR to your fork that restore the "old way" using `python setup.py install` instead of `pip`. That seems to work in all platforms. I know it sounds like a defeat, but at least we can get `4.1.1` out while we fix this.

**Comment by minrk** [comment]
Merged your PR. I'll open a new PR switching back to pip as soon as this works and can be merged.

**Comment by minrk** [comment]
Working, thanks @ocefpaf!

**Pull request merged**